src/liger_kernel/ops/cross_entropy.py

```diff
@@ -285,7 +285,11 @@ def cross_entropy_forward(
         num_warps=32 if not is_hip() else 16,
     )
 
-    loss = torch.sum(loss_1d)
+    if reduction == "none":
+        loss = loss_1d
+    else:
+        loss = torch.sum(loss_1d)
+
     if return_z_loss == _TRUE.value:
         z_loss = torch.sum(z_loss_1d)
```

Comment:
Let's add none reduction option for z_loss as well

Reply:
Thanks for pointing it out, just updated, pls review.